cpp/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@
 #=============================================================================
 cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
 
-project(CUDA_DATAFRAME VERSION 0.7.2 LANGUAGES C CXX CUDA)
+project(CUDA_DATAFRAME VERSION 0.8.0 LANGUAGES C CXX CUDA)
```

Comment:
Not sure why branch-0.9's CMakeLists.txt has cudf version as 0.7.2.

I'd think it should be 0.9.0 now. @raydouglass can you confirm?